mlir/utils/jenkins/Jenkinsfile

```diff
@@ -297,7 +311,7 @@ pipeline {
 
         // choose the codepath for testing
         choice(name: 'codepath',
-               choices: ['default', 'mfma', 'navi21', 'vanilla'],
+               choices: ['default', 'mfma', 'navi21', 'navi3x', 'vanilla'],
                description: 'Choose the codepath to test')
         // option to disable navi21 cells in case nodes are offline
         booleanParam(name: 'disableNavi21', defaultValue: false,
```

Comment:
Add an option to `disableNavi31`, and this PR should be ready to merge.

You may want to let the team know:
 - What does Navi3x node related issues look like
 - How do they manually kick a run skipping Navi3x nodes when that happens